transforms/templateDirectives/multipleTemplateIfTrueIfFalse.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { walkParse5Ast, serializeParse5Ast } from '../parse5Utils'
+
+const isIfTrueIfFalseAttr = (attr) => attr.name === 'if:true' || attr.name === 'if:false'
+
+const removeInvalidIfTrueIfFalseAttributes = (node, ctx) => {
+  if (node.attrs?.some(isIfTrueIfFalseAttr)) {
+    // In LWC templates only the first if:true/if:false is processed
+    // Keep the first occurrence of if:true/if:false and discard the rest
+    const idx = node.attrs.findIndex(isIfTrueIfFalseAttr)
+    node.attrs = node.attrs.reduce((acc, curr, index) => {
+      if (!isIfTrueIfFalseAttr(curr) || index === idx) {
+        acc.push(curr)
+      }
+      return acc
+    }, [])
```

Comment:
I don't think you need a `reduce` here – a `filter` would be more readable and simpler.